remo/sdk.py

```diff
@@ -170,10 +169,10 @@ def list_datasets(self) -> List[Dataset]:
         Returns:
             List[:class:`remo.Dataset`]
         """
-        resp = self.api.list_datasets()
+        json_data = self.api.list_datasets()
         return [
-            Dataset(self, id=dataset['id'], name=dataset['name'], quantity=dataset['quantity'])
-            for dataset in resp.get('results', [])
+            Dataset(self, **ds_item)
```

Comment:
@vmanilo I'd make the syntax more explicit here, for readability. 
I have no idea what's going on :D